### ISSUE TITLE
Error while using check_metric() function

### ISSUE THREAD

**Issue by anoopboseml** [pycaret/pycaret#718]
I am trying to use the check_metric() function to get the accuracy score for my model. But, I am getting an error that states this:

'<' not supported between instances of 'str' and 'int'

I have double checked everything in my code and everything appears to be ok from my end. My actual and prediction value columns are both numeric and not string. 

Note: This is a binary classification problem.

![part0](https://user-images.githubusercontent.com/63749480/95678658-a21d9580-0beb-11eb-99fc-bda3ce1cc223.PNG)

![problem1](https://user-images.githubusercontent.com/63749480/95678718-13f5df00-0bec-11eb-95ef-d559031a5267.PNG)

**Comment by anoopboseml** [comment]
OK, kind of found a workaround to the problem with PyCaret as well as SKLearn but they aren't very elegant. It appears that the problem is with the predict_model() function. The labels it produces are strings instead of integers. 

I got 2 solutions:

**Solution 1:**

from pycaret.utils import check_metric
check_metric(unseen_predictions.Outcome, unseen_predictions['Label'].astype(int), 'Accuracy')

**Solution 2:**

from sklearn.metrics import accuracy_score
accuracy_score(unseen_predictions['Outcome'], unseen_predictions['Label'].astype(int))

Both produce the same result in terms of value. 

It would be great if automatic conversion of 'Label' column to string can be stopped or atleast have an option to manage the output while using the predict_model() function.

**Comment by Yard1** [comment]
This will be fixed in the next update.

**Comment by pycaret** [comment]
@anoopboseml 

This issue has been fixed in the `pycaret-nightly` release. Please install pycaret-nightly using `pip install pycaret-nightly`. Feel free to open a new issue, in case the problem persists. This fix will be published to PyPi as `pycaret==2.2.0` on October 28, 2020.
